lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -40,6 +40,15 @@ function subscribeEvents() {
     const label = args !== null ? '#' + args.attr('id') : '';
     ga('send', 'event', 'Modal', 'Close', label);
   });
+
+  // Signup button on pitch page
+  const isPitchPage = $('.pitch').length > 0;
+  if (isPitchPage) {
+    // This applies to both authenticated & anonymous users. It also covers both modal register & login.
+    $('.header__signup form, #user-login-form, #user-register-form').on('submit', () => {
```

Comment:
Have you tested that `#user-register-form` doesn't trigger this analytics call if there are form validation errors? That may have to be a second event listener on the `Validation:Submitted` event so we don't get false positives from that form (see example usage above, `args` is the form ID).